packages/sku/src/lib/program/commands/start/start.action.ts

```diff
@@ -122,60 +123,57 @@ export const startAction = async ({
         devServerMiddleware(app);
       }
 
-      middlewares.push(
-        /** @type {import("express").RequestHandler} */
-        (req, res, next) => {
-          const matchingSiteName = getSiteForHost(req.hostname);
-
-          const matchingRoute = routes.find(({ route, siteIndex }) => {
-            if (
-              typeof siteIndex === 'number' &&
-              matchingSiteName !== sites[siteIndex].name
-            ) {
-              return false;
-            }
-
-            return routeMatcher(route)(req.path);
-          });
-
-          if (!matchingRoute) {
-            return next();
-          }
-
-          let chosenLanguage;
-
-          try {
-            chosenLanguage = getLanguageFromRoute(req, matchingRoute);
-          } catch (e) {
-            return res.status(500).send(
-              exceptionFormatter(e, {
+      middlewares.push(((req, res, next) => {
+        const matchingSiteName = getSiteForHost(req.hostname, undefined);
+
+        const matchingRoute = routes.find(({ route }) =>
+          // Comments on this? SiteIndex does not exist on the route type. I am unaware if this is irrelevant now.
```

Comment:
It should exist here. I think the removal of [these types](https://github.com/seek-oss/sku/pull/1137/files#diff-8b865db76e7386b701375c086a35411fbd382810b5792995a6120cba8ff012cfL71-R89) made it disappear. I think the type changes during the mutation of `normalizedRoutes` inside the `forEach`, which isn't picked up by typescript. `normalizedRoutes` might need an explicit type still.